pyproject.toml

```diff
@@ -8,6 +8,7 @@ authors = ["Pankaj Gupta <pankaj@baseten.co>", "Phil Howes <phil@baseten.co>"]
 include = ["*.txt", "*.Dockerfile", "*.md"]
 repository = "https://github.com/basetenlabs/truss"
 keywords = ["MLOps", "AI", "Model Serving", "Model Deployment", "Machine Learning"]
+packages = [{include = "truss", format = "wheel"}]
```

Comment:
so that we can `pip install /path/to/truss_package` which is now part of the base Dockerfile. This lets us build a wheel on the fly without installing poetry.